### PR TITLE
Add `init` like behavior.

### PR DESCRIPTION
**Pull request by taurr**
Relates to #193

### CODE REVIEW

**Review by sassman — APPROVED** [review]

**Comment by sassman** [comment]
could you pls rebase this branche?

File: README.md [hunk 1]
```diff
@@ -72,6 +74,16 @@ cargo generate --git https://github.com/githubusername/mytemplate.git <relative-
...
 
+#### Generating into current dir
+
+If the user wants to generate a template straight into the current folder, without creating a subfolder for the contents and without attempting to initialize a `.git` repo or similar, the `--init` flag can be used.
+
+```sh
+cargo generate --init --git https://github.com/githubusername/mytemplate.git
+```
+
+> NOTE: `cargo-generate` will not allow any existing files to be overwritten and will fail to generate any files should there be any conflicts.
```
[the remark this answers is not in the document]
Comment (taurr) [reply README.md:85]: I don't know about that - i kinda conflicts with the name resolution.

For now, it just fails and prints out the first file that conflicts. At least the user won't trash her/his workspace by accident.

**Review by sassman — APPROVED** [review]

**Comment by sassman** [comment]
Sorry, needs rebasing again

**Comment by taurr** [comment]
@sassman PRs are rebased and ready.

**Pull request merged**